### DISCUSSION TITLE
Media Lib. Pro TEMP upload

### DISCUSSION THREAD

**Discussion by faxunil** [spatie/laravel-medialibrary#2578] [Q&A]
Hi!
I have a working Vue app. 
Media is also uploaded file and SQL is present.
But If I want to join the uploaded file to my model I'm gettin UUID error.
After I have created the Event wanted to join the uploaded file
$event->addFromMediaLibraryRequest($request->get('media'))->toMediaCollection('tmp-xls');

"message": "invalid uuid",
    "exception": "Exception",
    "file": "/Users/gergelymac/gitrepos/cascade/vendor/spatie/laravel-medialibrary-pro/src/Dto/PendingMediaItem.php",
    "line": 41,

Any Idea?
Br.
Gergely

**Comment by ndberg** [comment]
For future Googler's I had this issue because I forgot to add the MediaCollection to my model:
```php
    public function registerMediaCollections() : void
    {
        $this->addMediaCollection('images');
    }
```

Which must match the name in:
```php
$property
    ->syncFromMediaLibraryRequest($request->media)
    ->toMediaCollection('images');
```
